### PR TITLE
refactoring: migrate get all events api

### PR DESCRIPTION
**Pull request by valerydluski**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [x] I followed naming convention rules

---

#### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [x] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

_Describe the source of requirement, like related issue link._

#### 💡 Background and solution

_Describe the big picture of your changes here_

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

File: nestjs/src/events/dto/event.dto.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  name: string;
+
+  @ApiProperty({ nullable: true, type: String })
+  descriptionUrl: string | null;
+
+  @ApiProperty({ nullable: true, type: String })
+  description: string | null;
+
+  @ApiProperty({ enum: EventType })
+  type: EventType;
```
Comment (apalchys) [inline nestjs/src/events/dto/event.dto.ts:29]: make all public

Comment (valerydluski) [reply nestjs/src/events/dto/event.dto.ts]: fixed thanks

File: server/src/models/event.ts [hunk 2]
```diff
@@ -16,10 +17,13 @@ export class Event {
   @PrimaryGeneratedColumn() id: number;
 
   @CreateDateColumn()
-  createdDate: number;
+  createdDate: string;
```
Comment (aaliakseyenka) [inline server/src/models/event.ts:20]: consider checking the proper type: based on the docs it should be `Date`

**Review by aaliakseyenka — APPROVED** [review]
👍

File: server/src/models/event.ts (hunk 2 above)
Comment (valerydluski) [reply server/src/models/event.ts]: I have fixed all the comments and am very grateful for the review

**Review by nataliagulko — APPROVED** [review]

**Pull request merged**